train.py

```diff
@@ -85,12 +84,13 @@ def run_eval(
     Returns:
         Validation loss
     """
-
+    torch.inference_mode(mode=True)
     with torch.no_grad():
```

Comment:
`no_grad` is probably redundant now, but we can leave it in there. 

I remember we had some issues with inference_mode a while back. Was it a necessary change? Otherwise, i'd stick with the old solution.

Reply:
Thanks for the catch, inference mode can probably be removed. 
Slipped in from https://github.com/h2oai/h2o-llmstudio/blob/main/train.py#L102 where it is actually not needed.